packages/flutter/test/widgets/semantics_test.dart

```diff
@@ -631,8 +632,50 @@ void main() {
     );
 
     expect(semantics, hasSemantics(expectedSemantics, ignoreId: true));
+
+    await tester.pumpWidget(
+        Semantics(
+          key: const Key('a'),
+          container: true,
+          explicitChildNodes: true,
+          // flags
```

Comment:
Is this comment necessary?

Reply:
Are you referring to `//flags`? This was already present in the earlier definitions in this test function, so it was copied to the modified instance created here.